### DISCUSSION TITLE
[Bug v2.0.0-alpha.6] Animated gif images are static when cache version is served

### DISCUSSION THREAD

**Discussion by slowscape** [automadcms/discussions#18] [General]
Title says it all, no movement in animated gif images when the cache version is served.

**Comment by marcantondahmen** [comment]
Hi, this is the expected behavior. Resizing animated gifs is quite a task for a web server.

However, the way previews or thumbnails are implemented is up to the theme developer. You can easily create a theme that doesn't resize images out of the box. It is just how it works in the standard themes.
